components/aws/sagemaker/TrainingJob/README.md

```diff
@@ -0,0 +1,99 @@
+# SageMaker Training Kubeflow Pipelines component
+
+Component to create [SageMaker Training jobs](https://docs.aws.amazon.com/sagemaker/latest/dg/how-it-works-training.html) in a Kubeflow Pipelines workflow.
+
+| :warning: WARNING: This component is in Preview Release Phase. It is not GA |
+| --- |
+
+## Overview
+
+The Amazon SageMaker components for Kubeflow Pipelines version 1(v1.1.x or below) uses [Boto3 (AWS SDK for Python)](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker.html) as the backend to create and manage resources on SageMaker. SageMaker components version 2(v2.0.0-alpha1 or above) uses the [ACK Service Controller for SageMaker](https://github.com/aws-controllers-k8s/sagemaker-controller) to do the same. AWS introduced [ACK](https://aws-controllers-k8s.github.io/community/) to facilitate a Kubernetes-native way of managing AWS Cloud resources. ACK includes a set of AWS service-specific controllers, one of which is the SageMaker controller. The SageMaker controller makes it easier for machine learning developers and data scientists who use Kubernetes as their control plane to train, tune, and deploy machine learning models in Amazon SageMaker.
```

Comment:
Can we link to the train/README.

Reply:
Changed the title and in the main README called out v1 and v2 clearly. I think it should be good for now, wdyt